packages/flame_test/lib/src/flame_test.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flame/game.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:meta/meta.dart';
+
+extension FlameFinds on CommonFinders {
+  Finder byGame<T extends Game>() {
+    return find.byWidgetPredicate(
+      (widget) => widget is GameWidget<T>,
+    );
+  }
+}
+
+typedef GameCreateFunction<T extends Game> = T Function();
+typedef VerifyFunction<T extends Game> = void Function(T);
+
+/// Creates a [Game] specific test case with given [description].
+///
+/// Use [createGame] to create your game instance.
+///
+/// Use [verify] closure to make verifications/assertions.
+@isTest
+void flameTest<T extends Game>(
+  String description, {
+  required GameCreateFunction<T> createGame,
+  required VerifyFunction<T>? verify,
```

Comment:
If `verify` is nullable, then why is it required?

Reply:
I forgot to removed the required, removed now